File: 2 - Operators/Solution.swift
import Foundation

let mealCost = Double(readLine()!)!
let tipPercent = Int(readLine()!)!
let taxPercent = Int(readLine()!)!

let tip = Double(tipPercent) * mealCost / 100;
let tax = Double(taxPercent) * mealCost / 100;

let totalCost = round(tip + tax + mealCost);
print("The total meal cost is \(Int(totalCost)) dollars.")

File: 2 - Operators/Solution.fs
open System

[<EntryPoint>]
let main argv = 
    let mealCost = Console.ReadLine() |> double
    let tipPercent = Console.ReadLine() |> int
    let taxPercent = Console.ReadLine() |> int
    let tip = float tipPercent * mealCost / 100.0
    let tax = float taxPercent * mealCost / 100.0
    let totalCost = Math.Round(tip + tax + mealCost) |> int
    printfn "The total meal cost is %i dollars." totalCost
    0
